src/Server.cpp

```diff
@@ -0,0 +1,81 @@
+// Copyright (c) 2021 Pei-Ru Wang, Cheryl Huang, Yuan Zhou. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+/**
+ * Project MyHttpServer
+ * @author Pei-Ru Wang, Cheryl Huang, Yuan Zhou
+ * @version 0.0.1
+ */
+
+#include "Server.h"
+
+#include <boost/asio.hpp>
+#include <boost/bind.hpp>
+#include <boost/enable_shared_from_this.hpp>
+#include <boost/lexical_cast.hpp>
+#include <boost/shared_ptr.hpp>
+#include <thread>
```

Comment:
add new line to separate C++ standard library thread and 3rd party library boost

Reply:
This is a reasonable reviw. You can cite the style guide as well: https://google.github.io/styleguide/cppguide.html#Names_and_Order_of_Includes